OSDOCS-13815: Documented the 4.17.23 z-stream RNs (#91256)
openshift/openshift-docs: release_notes/ocp-4-17-release-notes.adoc

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.17.23 --pullspecs
----

////
[id="ocp-4-17-23-known-issues_{context}"]
==== Known issues

* Previously for a cluster that runs on {rh-openstack-first}, if the router and virtual machines (VMs) existed on different compute nodes, the distribution of floating IP (FIP) addresses and virtual IP (VIP) addresses on control plane nodes was impacted. //issues.redhat.com/browse/OCPBUGS-52160
////

[id="ocp-4-17-23-bug-fixes_{context}"]
==== Bug fixes

* Previously, the Operator Marketplace and the {olm-first} used an older version, v1.24, of the `pod-security.kubernetes.io/` label. With this release, the namespace where Operator Marketplace is deployed now uses the Pod Security Admission (PSA) label marked as `latest`. (link:https://issues.redhat.com/browse/OCPBUGS-53283[*OCPBUGS-53283*]) 

> * Previously, when you ran the `openshift-install agent create pxe-files` command, the command created temporary directories in `/tmp/agent` and the command did not remove these directories upon command completion. With this release, the command now removes the directories when the command nears completion, so no manual deletion of directories is necessary. (link:https://issues.redhat.com/browse/OCPBUGS-52961[*OCPBUGS-52961*])

Reviewer: ```suggestion
* Previously, the `openshift-install agent create pxe-files` command created temporary directories in `/tmp/agent` and did not remove these directories upon command completion. With this release, the command now removes the directories upon completion, so no manual deletion of directories is necessary. (link:https://issues.redhat.com/browse/OCPBUGS-52961[*OCPBUGS-52961*])
```

IMO _when you ran_ is intended

Author: :heavy_check_mark:

> * Previously, a code migration operation caused external labels to not be processed correctly on the *Alert detail* page on the *Administrator perspective* of the web console. These external labels are required to prevent silenced alert notifications from getting added to notification bell icon. Because the *Alert detail* page did not handle external labels correctly, the notification bell provided links to these *Alert detail* pages that generated a `no matching alerts found` message when you clicked on a link. With this release, the *Alert detail* page accepts external labels so clicking on an alert in the notification bell links to the correct *Alert detail* page. (link:https://issues.redhat.com/browse/OCPBUGS-51117[*OCPBUGS-51117*])

Reviewer: ```suggestion
* Previously, a code migration operation failed to process external labels correctly on the *Alert detail* page on the *Administrator perspective* of the web console. These external labels are required to prevent silenced alert notifications from getting added to notification bell icon. Because the *Alert detail* page did not handle external labels correctly, the notification bell provided links to these *Alert detail* pages that generated a `no matching alerts found` message when you clicked on a link. With this release, the *Alert detail* page accepts external labels so clicking on an alert in the notification bell links to the correct *Alert detail* page. (link:https://issues.redhat.com/browse/OCPBUGS-51117[*OCPBUGS-51117*])
```

Author: :heavy_check_mark:

> * Previously, the Single Root I/O Virtualization (SR-IOV)  network config daemon unbinded network drivers from the physical function (PF) interface instead of unbinding the drivers from the virtual function (VF) interface when SR-IOV was configured with an InfiniBand (IB) type. This unbinding workflow removed the IB interface from the node, and this sitaution made the IB interface non-functional. With this release, a fix to the SR-IOV network config daemon ensures that the IB interface remains functional when it correctly unbinds the VF network interface. Additionally, the SR-IOV Network Operator targets the network drivers of a VF interface instead of the PF interface when configuring SR-IOV with an InfiniBand (IB) type. (link:https://issues.redhat.com/browse/OCPBUGS-53254[*OCPBUGS-53254*])

Reviewer: ```suggestion
* Previously, the Single Root I/O Virtualization (SR-IOV) network config daemon unbinded network drivers from the physical function (PF) interface instead of unbinding the drivers from the virtual function (VF) interface when SR-IOV was configured with an InfiniBand (IB) type. This unbinding workflow removed the IB interface from the node, and this sitaution made the IB interface non-functional. With this release, a fix to the SR-IOV network config daemon ensures that the IB interface remains functional when it correctly unbinds the VF network interface. Additionally, the SR-IOV Network Operator targets the network drivers of a VF interface instead of the PF interface when configuring SR-IOV with an IB type. (link:https://issues.redhat.com/browse/OCPBUGS-53254[*OCPBUGS-53254*])
```

IB is already expanded earlier.

Author: :heavy_check_mark:
